LevelUp.Pos.ProposedOrders.Tests/Mocks/PointOfSale.cs

```diff
@@ -35,8 +37,15 @@ internal class Check
         /// </summary>
         /// <param name="total">The intial amount due on the check including tax, in cents.</param>
         /// <param name="tax">The initial tax due on the check, in cents.</param>
-        public Check(int total, int tax)
+        public PointOfSale(int total, int tax)
         {
+            if (total == tax)
+            {
+                throw new Exception($"The total must be greater than tax (a subtotal amount must exist). If the intent " +
```

Comment:
Perhaps looking mostly for feedback on this.  Any way the error message could be improved?

Reply:
I think this message makes sense.